modules/dns_dnscmd/dns_dnscmd_main.rb

```diff
@@ -115,8 +123,17 @@ def enum_zones
         next unless line =~  / Primary /
         zones << line.sub(/^ +/, '').sub(/ +.*$/, '').chomp("\n")
       end
-      logger.debug "Enumerated dns zones: #{zones}"
+      logger.debug "Enumerated dns authoritative dns zones: #{zones}"
       zones
     end
+
+    def get_ptr_record(ip)
+      begin
```

Comment:
You can remove explicit try-rescue boundaries in the context of a method definition:

```
def get_ptr_record(ip)
  ipaddr = IPAddr.new(ip)
  ipaddr.reverse
rescue
  raise Proxy::Dns::Error.new("Not an IP address: '#{ip}'")
end
```